pkg/gui/custom_commands.go

```diff
@@ -52,12 +52,12 @@ func (gui *Gui) createCommandMenu(customCommands []config.CustomCommand, command
 			return option.customCommand.InternalFunction()
 		}
 
+		if option.invokeShell {
+			option.command = gui.OSCommand.NewCommandStringWithShell(option.command)
+		}
+
 		// if we have a command for attaching, we attach and return the subprocess error
 		if option.customCommand.Attach {
-			if option.invokeShell {
-				return gui.runSubprocess(gui.OSCommand.ExecutableFromStringWithShell(option.command))
-			}
-
 			return gui.runSubprocess(gui.OSCommand.ExecutableFromString(option.command))
 		}
```

Comment:
Since we moved the fix for a user config level, i think it makes sense to allow users to run invoked shell cmds on attach true and false.